projects/UMissouri_Nada.yaml

```
@@ -0,0 +1,14 @@
+Description: >- 
+Running segmentation of 100 subjects through Freesurfer. Comparing the volumetric measurements of the 
+midbrain between the disease group (PS) and control group.
+PSP is a movement disorder characterized by atrophy of the midbrain.
+We investigate quantification of the midbrain volume to early predict the disease.
```

```suggestion
Description: >- 
  Running segmentation of 100 subjects through Freesurfer. Comparing the volumetric 
  measurements of the 
  midbrain between the disease group (PS) and control group.
  PSP is a movement disorder characterized by atrophy of the midbrain.
  We investigate quantification of the midbrain volume to early predict the disease.
```
The multiline string needs to be indented, otherwise the program can't tell where it ends.